### DISCUSSION TITLE
Potential reassignment of modeling task(s)

### DISCUSSION THREAD

**Discussion by ajshajib** [AstroBridge/BDLensing#103] [Announcements]
Whereas some participants are advancing toward finishing modeling the systems they were assigned, a few lens systems didn't receive any updates from their assigned modelers (for some cases, the associated issues haven't been created at all). Once a participant is done with their current model and looking for a new system, such "abandoned" systems will be reassigned to the participant seeking to work on a new one. 

**If you want to avoid your originally assigned system from getting reassigned, please start actively working on them and share your progress through pull requests ASAP.** If I look at [this page](https://github.com/ajshajib/BDLensing/tree/main/lens_systems), some systems didn't receive any updates in the last two months, which indicates that they have been abandoned by the original assignee.

**Comment by Rajit13** [comment]
Who is working on `DESIJ1709+3154`? I am seeing no progress in the past 2 months. I'll take it if no one else does.

**Reply by ajshajib** [reply]
DESIJ1709+3154 was assigned to @Sinthia666. She's working on it based on #97.

**Reply by TwistNemo** [reply]
I am working on "DESIJ1709+3154".

**Comment by ajshajib** [comment]
@Rajit13, can you kindly check for which systems the associated issues are not created yet?

**Reply by Rajit13** [reply]
No issue has been created for `DESIJ0132-1600` and `DESIJ1539+1650`.

**Reply by ajshajib** [reply]
`DESIJ0132-1600` was assigned to @anikbiswas730 on May 13, and `DESIJ1539+1650` was assigned to @CHOC4HOL1C on June 3.

@anikbiswas730, have you made any progress on the model? If so, please share your work ASAP through a PR. Also, what is your plan to meet future deadlines for each milestone or issue?

**Reply by anikbiswas730** [reply]
Yes vai  I have completed the masking and my best chi square is 1.5 till now , I thought I had to upload after completing everything. I'm so sorry for my ignorance

**Comment by ajshajib** [comment]
Since @Rajit13 and @rafee-adnan are done (or almost done) with their lens systems, I suggest you help with some of the other more difficult ones. In that case, you can start with the current/latest notebook on that lens system and then experiment with different settings to see if you can improve the model. If you find a suitable model, please send a PR. Please coordinate with the initially assigned modeler while working on these systems so that conflicting PRs can be avoided down the line.

I suggest @rafee-adnan help with @numjahan's system and @Rajit13 help with @SulymanHossain's system. Note that this is **not a re-assignment**; we are just doubling hands to work on the more complicated systems here. 

@numjahan and @SulymanHossain, one way to share your current notebooks with @rafee-adnan and @Rajit13 would be to commit your current notebooks to your own forks.
